kafka/consumer_test.go

```diff
@@ -65,6 +64,20 @@ func TestConsumerAPIs(t *testing.T) {
 		t.Errorf("Unsubscribe failed: %s", err)
 	}
 
+	topic := "gotest"
+	stored, err := c.OffsetsStore([]TopicPartition{{Topic: &topic, Partition: 0, Offset: 1}})
+	if err != nil && err.(Error).Code() != ErrUnknownPartition {
+		t.Errorf("OffsetsStore() failed: %s", err)
+		toppar := stored[0]
+		if toppar.Error.(Error).Code() == ErrUnknownPartition {
+			t.Errorf("OffsetsStore() TopicPartition error: %s", toppar.Error)
+		}
+	}
+	stored, err = c.OffsetsStore(nil)
```

Comment:
Right, the data type allows it, but is there a point for the StoreOffsets API to allow it? Passing a nil slice should be considered a usage error and not something we need to explicitly handle (I do agree that error handling is necessary, but parameter checking in APIs is not as fruitful as it seems IMHO unless there is a security concern to it).